### PR TITLE
Re-introduce property 9

### PR DESCRIPTION
**Pull request by blt**
### What does this PR do?

This commit re-introduces property 9, bounding the size of a rotated File. We
allow the maximum size to incorporate an additional tick but no more than that.

### Related issues

REF SMPTNG-517

### CODE REVIEW

**Comment by blt** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/DataDog/lading/1093?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1093** <a href="https://app.graphite.dev/github/pr/DataDog/lading/1093?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1092** <a href="https://app.graphite.dev/github/pr/DataDog/lading/1092?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @blt and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lading/src/generator/file_gen/logrotate_fs/model.rs [hunk 1]
```diff
@@ -1216,44 +1209,29 @@ mod test {
...
-        //             file.bytes_written >= min_size && file.bytes_written < max_size,
-        //             "Rotated file size {bytes_written} not in expected range [{min_size}, {max_size})",
-        //             bytes_written = file.bytes_written
-        //         );
-        //     }
-        // }
+        // Property 8: Rotated files have bytes_written within acceptable range
+        //
+        // For a rotated file (read_only == true), bytes_written should be
+        // within (max_bytes_per_file - bytes_per_tick) <= bytes_written <
```
Comment (goxberry) [inline lading/src/generator/file_gen/logrotate_fs/model.rs:1215]: The lower bound asserted in the code below appears to be tighter than the lower bound asserted in this comment. Should these two lower bounds be identical?

Comment (blt) [reply lading/src/generator/file_gen/logrotate_fs/model.rs:1215]: Oops I neglected to update this in the comment. Good eye.

Comment (blt) [reply lading/src/generator/file_gen/logrotate_fs/model.rs]: Addressed in [7d4f26d](https://github.com/DataDog/lading/pull/1093/commits/7d4f26d046c2aa6a6614577c79eefcc4202a538f).

**Review by goxberry — APPROVED** [review]

**Comment by blt** [comment]
### Merge activity

* **Nov 5, 7:28 PM EST**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/DataDog/lading/1093).

**Pull request merged**